app/grants/templates/grants/explorer.html

```diff
@@ -117,27 +121,31 @@
                         <template v-for="(round_type, key) in round_types">
                           <div
                             class="d-flex text-capitalize cursor-pointer bg-violet-nav-item"
-                            @click.stop="selectRoundType(round_type)"
+                            @click.stop="isSelectedRoundType(round_type, key) && !closed ? closed = true : selectRoundType(round_type)"
                           >
-                            <div :class="`w-100 h-100 line-clamp-1 line-height-4 text-grey-500 px-2 py-1 ${ (params.round_type && params.round_type == round_type) || (!params.round_type && key == 0) ? 'bg-violet-nav' : '' }`">
+                            <div :class="`d-flex justify-content-between align-items-center w-100 h-100 line-clamp-1 line-height-4 text-grey-500 px-4 py-2 ${ isSelectedRoundType(round_type, key) ? 'bg-violet-nav' : '' }`">
                               [[ round_type ]]
+                              <i :class="`d-md-none ${ isSelectedRoundType(round_type, key) && !closed ? 'fas fa-chevron-up' : 'fas fa-chevron-down' }`"></i>
```

Comment:
For:
GIVEN I am viewing the grants explorer on desktop during an active grants round
WHEN I tap the grant round filter
THEN The carat symbol should not appear

Will the carat hide on desktop? Please disregard if so  :). I just didn't see any logic conditionally display it

Reply:
@schultztimothy Yep, it will hide. The reason why there's no conditional logic handling that is because the mobile and desktop views for this particular menu are two different templates. The mobile view is lines `121-142` and the desktop view is lines `146-153`

Reply:
Awesome thanks for clarifying! LGTM!